### DISCUSSION TITLE
macOS 12 install doesn't pass 0%

### DISCUSSION THREAD

**Discussion by Aem1024** [utmapp/UTM#3566] [UTM Help (macOS)] [answered]
**Describe the issue**
macOS sticks on 0% when installing using apple virtualization.

**Configuration**
UTM Version 3.0.3 & 3.0.4
macOS Version 12.0.1 (21A559)
Apple M1

**Comment by osy** [comment, marked as answer]
1) how long did you wait
2) can you make sure you’re not out of disk space

**Reply by Aem1024** [reply]
i waited about 20 minutes, and i have 150gb free

**Reply by osy** [reply]
can you update your system to 12.2?

**Reply by Aem1024** [reply]
i'll update now

**Reply by Aem1024** [reply]
updated and it's working.

**Comment by MayCXC** [comment]
getting same thing on osx 12.4 and UTM 3.2.4 followed by "Installation failed"

EDIT: this was due to reducing the default memory capacity.
